config/jobservice/config.yaml

```diff
@@ -9,6 +9,7 @@ databaseType: "postgres"
 # storage file when using database type 'sqlite'
 databasePath: "/var/jobservice.db"
 # Connection details when using database type 'postgres'
+# to be replaced with databaseConfig
```

Comment:
I think we should try to be backwards compatible with this config key -- there are a lot configuration repos out there that I wouldn't want to track down!

Reply:
I agree, and instead of immediately switching to a new field and removing the existing `postgresConfig` field, we can have them both temporarily. if a user sets `postgresConfig` we can then display a warning message saying that this field is getting deprecated in future releases and that the new field that's meant to provide database configuration to job service is `databaseConfig`. 
this logic is now available in my latest commit ^^